index.html

```diff
@@ -336,9 +352,11 @@
             });
     }
 
-    const loadContent = async (hash) => {
-        console.log("loadContent");
+    const loadContent = async () => {
+        cwd = window.location.hash.substr(0, window.location.hash.lastIndexOf("/")) + "/";
```

Comment:
this line was the core fix.. thanks @arnaudsm

Reply:
Thanks for your help, sorry it took so long :pray: